website/docs/i18n/i18n-tutorial.mdx

```diff
@@ -390,11 +390,43 @@ Official Docusaurus content plugins extensively use Markdown/MDX files and allow
 
 Copy your docs Markdown files from `docs/` to `i18n/fr/docusaurus-plugin-content-docs/current`, and translate them:
 
+<Tabs>
+<TabItem value="Linux">
+
 ```bash
-mkdir -p i18n/fr/docusaurus-plugin-content-docs/current
-cp -r docs/** i18n/fr/docusaurus-plugin-content-docs/current
+mkdir -p i18n\fr\docusaurus-plugin-content-docs\current
+cp -r docs\** i18n\fr\docusaurus-plugin-content-docs\current
```

Comment:
this command is working fine if you on windows as long as you use `powershell` instead of `cmd`

![image](https://user-images.githubusercontent.com/625469/235893199-8c6c858f-def4-47b9-96c1-9027335b2174.png)

Reply:
I agree with Sebastién that a single, cross-platform command would be best. See here: https://github.com/facebook/docusaurus/pull/8704#pullrequestreview-1320448975